File: mongodb_atlas_test.java
package net.codejava.mongodb;
 
import java.net.UnknownHostException;
import java.util.List;
import java.util.Set;
 
import com.mongodb.DB;
import com.mongodb.MongoClient;
 
public class JavaMongoDBConnection {
 
    public static void main(String[] args) {
        try {

            MongoClientURI uri = "mongodb+srv://user1:user1pw@msmongotestcluster.x30yk.mongodb.net/mytestdb?retryWrites=true\\&w=majority\\&authMechanism=SCRAM-SHA-1\\&useNewUrlParser=true\\&useUnifiedTopology=true";
             
            MongoClient mongoClient = new MongoClient(uri);
             
            List<String> databases = mongoClient.getDatabaseNames();
             
            for (String dbName : databases) {
                System.out.println("- Database: " + dbName);
                 
                DB db = mongoClient.getDB(dbName);
                 
                Set<String> collections = db.getCollectionNames();
                for (String colName : collections) {
                    System.out.println("\t + Collection: " + colName);
                }
            }
             
            mongoClient.close();
             
        } catch (UnknownHostException ex) {
            ex.printStackTrace();
        }
         
    }
}

File: mongodb_atlas_test.py
from pymongo import MongoClient
# pprint library is used to make the output look more pretty
from pprint import pprint
# connect to MongoDB, change the << MONGODB URL >> to reflect your own connection string
client = MongoClient("mongodb+srv://user1:user1pw@msmongo1.x30yk.mongodb.net/mytestdb?retryWrites=true&w=majority")
db=client['mytestdb']
# print the contents of collection
testCollectionDocs=db["testCollection"].find()
for doc in testCollectionDocs:
   pprint(doc)
